### PR TITLE
Rbug spellchek

### PR DESCRIPTION
**Pull request by ka1bi4**
I fixed some mistakes.

My branch is even with qgis:master 👼 

Goal:

Ticket(s): #

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/training_manual/processing/john_snow.rst [hunk 1]
```diff
@@ -10,13 +10,13 @@ First analysis example
...
+We will be using the well-known dataset that John Snow used in 1854, in his groundbreaking work (http://en.wikipedia.org/wiki/John_Snow_%28physician%29), and we will get some interesting results. The analysis of this dataset is pretty obvious and there is no need for sophisticated GIS techniques to end up with good results and conclusions, but it is a good way of showing how these spatial problems can be analyzed and solved by using different processing tools.
 
 The dataset contains shapefiles with cholera deaths and pump locations, and an OSM rendered map in TIFF format. Open the corresponding QGIS project for this lesson.
 
 .. image:: img/john_snow/project.png
 
-The first thing to do is to calculting the Voronoi diagram (a.k.a. Thyessen polygons) of the pumps layer, to get the influence zone of each pump. The *Voronoi Diagram* algorithm can be used for that.
+The first thing to do is to calculating the Voronoi diagram (a.k.a. Thyessen polygons) of the pumps layer, to get the influence zone of each pump. The *Voronoi Diagram* algorithm can be used for that.
```
Comment (havatv) [inline source/docs/training_manual/processing/john_snow.rst]:
```suggestion
The first thing to do is to calculating the Voronoi diagram (a.k.a. Thiessen polygons) of the pumps layer, to get the influence zone of each pump. The *Voronoi Diagram* algorithm can be used for that.
```

**Comment by ka1bi4** [comment]
:cry: omg, but why my branch is even

**Comment by DelazJ** [comment]
> My branch is even with qgis:master 

Yes, your master branch is even with qgis:master but not the branch you are using here. Your most recent commit in this branch is four days old, hence based on your outdated master. You could do a `git rebase master` to fix but there are some files that have substantially changed and would require more work to fix the conflict.
Imho, the wisest way to solve this is to manually report the changes here in a branch based on your master. Ie
```bash
# enable locally the branch
git checkout rbug-spellchek
# make it even with master, the hard way
# (you'll lose locally your work but no worries, they are online)
git reset --hard master
# modify locally the files, using the online version as reference
# once done, add your changes and create your commit
# your branch should now be one commit ahead master
# eg if you do "git branch --merged", master will be returned in the list
# push to GitHub
git push -f origin rbug-spellchek
```

File: source/docs/user_manual/preamble/contributors.rst [hunk 2]
```diff
@@ -32,7 +32,7 @@ and update the whole QGIS documentation.
 +--------------------+---------------------+----------------------+-----------------------+----------------------+
 | Tara Athan         | Radim Blazek        | K\. Koy              | Godofredo Contreras   | Martin Dobias        |
 +--------------------+---------------------+----------------------+-----------------------+----------------------+
-| Peter Ersts        | Anne Ghisla         | Stephan Holl         | N\. Horning           | Magnus Homann        |
+| Peter Ersts        | Anne Ghisla         | Stephan Holl         | N. Horning            | Magnus Homann        |
```
Comment (DelazJ) [inline source/docs/user_manual/preamble/contributors.rst]: I think This was right and shouldn't change (check eg how it renders in GitHub preview).
 "N." at the beginning of a new paragraph, and a cell can be assimilated to a new paragraph, means 14. (doing numbering). Here we really need to display the period hence the escape character.
```suggestion
| Peter Ersts        | Anne Ghisla         | Stephan Holl         | N\. Horning           | Magnus Homann        |
```

**Pull request closed without merging**